.bazelrc

```diff
@@ -99,6 +99,7 @@ build:windows --host_linkopt=/OPT:ICF
 build:windows --experimental_strict_action_env=true
 
 build:linux --config=posix
+build:linux --copt=-Wno-unknown-warning-option
```

Comment:
This is necessary since we are using gcc 9.4 and the subsequent two warning flags don't exist so the compiler gives an error.